install-onlyoffice.sh

```diff
@@ -166,22 +184,31 @@ install_version() {
 }
 
 install_x2t() {
-    ensure_command_available curl
-    ensure_command_available sha512sum
-    ensure_command_available unzip
```

Comment:
I moved these checks down to right before we use the relevant tools. I'd argue this is better style (keep related things closer to gether). This is also helpful so the nix build (which installs OnlyOffice in its own way) doesn't have to depend on curl. (see https://github.com/NixOS/nixpkgs/pull/365514)